k8s/secret.yml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)